src/java/org/apache/cassandra/config/AccordSpec.java

```diff
@@ -71,4 +73,62 @@ public enum TransactionalRangeMigration
     public TransactionalMode default_transactional_mode = TransactionalMode.off;
     public boolean ephemeralReadEnabled = false;
     public boolean state_cache_listener_jfr_enabled = true;
+    public final JournalSpec journal = new JournalSpec();
+
+    public static class JournalSpec implements Params
+    {
+        public int segmentSize = 32 << 20;
+        public FailurePolicy failurePolicy = FailurePolicy.STOP;
+        public FlushMode flushMode = FlushMode.BATCH;
+        public DurationSpec.IntMillisecondsBound flushPeriod; // pulls default from 'commitlog_sync_period'
+        public DurationSpec.IntMillisecondsBound periodicFlushLagBlock = new DurationSpec.IntMillisecondsBound("1500ms");
+
+        @Override
+        public int segmentSize()
+        {
+            return segmentSize;
+        }
+
+        @Override
+        public FailurePolicy failurePolicy()
+        {
+            return failurePolicy;
+        }
+
+        @Override
+        public FlushMode flushMode()
+        {
+            return flushMode;
+        }
+
+        @JsonIgnore
+        @Override
+        public int flushPeriodMillis()
+        {
+            return flushPeriod == null ? DatabaseDescriptor.getCommitLogSyncPeriod()
+                                       : flushPeriod.toMilliseconds();
+        }
+
+        @JsonIgnore
```

Comment:
I think this isn't needed as `get` is needed for it to be picked up... I was just trying to be explicit...